packages/site/src/content/ConfirmationModal/ConfirmationModal.props.json

```diff
@@ -185,4 +185,4 @@
       }
     }
   }
-]
```

Comment:
Not that it's a blocker, but this was also in the `InputFile` PR - do we know why this is happening? Is it a consequence from the generator?

Reply:
I think so 🤔